### ISSUE TITLE
Swapped values in the viewangletox array (angletoxTable.eps)

### ISSUE THREAD

**Issue by gianlucag** [fabiensanglard/gebbdoom#44]
On the angletoxTable.eps picture, shouldn't it be:
```
viewangletox[1028] = 319
viewangletox[3073] = 0
```

![angletox](https://user-images.githubusercontent.com/6398801/230571908-13ab7869-e615-4dd5-bfcd-2a84ade85b68.png)

**Comment by fabiensanglard** [comment]
Yes, there is a mistake in the drawing. Do you want to send a pull request :) ?

**Comment by gianlucag** [comment]
Ok. I didn't submit a pull req straight away because I'm not quite familiar with the .eps format. I guess I need to update the .svg as well...

**Comment by fabiensanglard** [comment]
Sorry this part of the build is a bit convoluted. O word with the SVG in
inkscape. Then I save the SVG but also Save copy to eps.

**Comment by gianlucag** [comment]
Done, I pushed the changes and submitted the PR. Please have a look, the .eps looks fine but I got lots of changes inside the file, just by swapping the two values of the array. I don't know if I did it right or not...
